docs/regional_coverage/statistics/energy-balance.md

```diff
@@ -2,20 +2,17 @@
 
 General Information 
 
-An energy balance provides a detailed and comprehensive overview of the energy situation in a specific country or region. Several pieces of information are contained:
+An energy balance provides a detailed and comprehensive overview of the energy situation in a specific country or region. The energy balance provides all relevant information to describe all end-use sectors (agriculture, commercial, industry, residential and transport) for an energy system model. Therefore, energy carriers, such as coal, oil and gas are covered in the energy statistics flow. Several pieces of information are contained:
```

Comment:
>Therefore, energy carriers, such as coal, oil and gas are covered in the energy statistics flow.

Please rephrase. "Therefore" seems out of place, while "energy statistics flow" is confusing.

Reply:
Done!